src/create-jobs.js

```diff
@@ -271,7 +282,7 @@ export default function (config) {
         .filter(function (job) {
             let keep = true;
             config.exclude.forEach(function (exclusion) {
-                keep = keep && ! excludeJob(job, exclusion);
+                keep = keep && ! excludeJob(job, exclusion, config.exclude_strict_matching);
```

Comment:
I think you want a `!` on the `config.exclude_strict_matching` here if set to true, it sets `strict_comparison` to true inside `excludeJob`, which is the opposite of the intent.

Reply:
Have to double check this again but I did some integration test on my local machine and that actually worked as intended.
However, with #83 and #62, this will become pretty easy.